docs/.vitepress/theme/custom.css

```diff
@@ -21,7 +22,22 @@
   --vp-c-brand-2: var(--vp-c-purple-2);
   --vp-c-brand-3: var(--vp-c-purple-3);
   --vp-c-brand-soft: var(--vp-c-purple-soft);
-  --vp-font-family-base: -apple-system, BlinkMacSystemFont, "avenir next", avenir, helvetica, "helvetica neue", ubuntu, roboto, noto, "segoe ui", arial, sans-serif;
+  --vp-font-family-base: Inter, -apple-system, BlinkMacSystemFont, "avenir next", avenir, helvetica, "helvetica neue",
```

Comment:
I didn't intend the change the base font. Only the content font. This changes the left nav and the ToC also, which was not the intent of this PR.

Reply:
OK I'll revert that. At this size the fonts are almost identical, I can only differentiate them with the inspector :)

Reply:
I don’t think it looks good to have a mix of Inter for the body and San Francisco for the sidebar. We should use the same sans-serif font everywhere.

Reply:
I will run it by design.

Reply:
Thanks @CobusT. I asked Kyna and Alex earlier this evening, and also about consolidating Framework’s accent colors (since we’re also using a blue and a red in addition to phosphate there). I put up https://github.com/observablehq/framework/pull/1591 for Framework and if there’s agreement there I expect we can propagate some of those changes to Plot. Like maybe the Plot should use phosphate for its accent color instead of purple, too…